deploy/helm/values-production.yaml

```diff
@@ -5,13 +5,20 @@
 replicaCount: 4
 
 ingress:
-  className: default
-  hosts:
-    - laa-hmrc-interface.cloud-platform.service.justice.gov.uk
+  enabled: false
+
+publicIngress:
+  enabled: true
+  className: default-non-prod
+  whitelist:
+    enabled: true
   annotations:
-    external-dns.alpha.kubernetes.io/set-identifier: "laa-hmrc-interface-laa-hmrc-interface-service-api-laa-hmrc-interface-production-green"
```

Comment:
is this not missing this line, that you have in staging below.

```
external-dns.alpha.kubernetes.io/set-identifier: "laa-hmrc-interface-public-laa-hmrc-interface-service-api"
```

and does it not need to be `...-production-green`